### PR TITLE
Luca6804/citrav2 doc

### PR DESCRIPTION
**Pull request by ldanzinger**
@maryharvey please review and merge if approved. We will need to cherry pick to main as well. If you are able to help with that, that would be great. Otherwise, I can assist next week.

### CODE REVIEW

File: uitools/toolkitcpp/cpp/Esri/ArcGISRuntime/Toolkit/BasemapGalleryController.cpp [hunk 1]
```diff
@@ -282,7 +282,7 @@ namespace Esri::ArcGISRuntime::Toolkit {
 
     The controller will automatically populate itself with the developer basemaps from AGOL.
```
Comment (maryharvey) [inline uitools/toolkitcpp/cpp/Esri/ArcGISRuntime/Toolkit/BasemapGalleryController.cpp:283]:
```suggestion
    The controller will automatically populate itself with the developer basemaps from ArcGIS Online.
```

File: uitools/toolkitwidgets/cpp/Esri/ArcGISRuntime/Toolkit/BasemapGalleryController.cpp [hunk 2]
```diff
@@ -282,7 +282,7 @@ namespace Esri::ArcGISRuntime::Toolkit {
 
     The controller will automatically populate itself with the developer basemaps from AGOL.
 
-     The given default basemaps require either an API key or named-user to be signed into the app.
+    The given default basemaps require either an access token or named-user to be signed into the app.
```
Comment (maryharvey) [inline uitools/toolkitwidgets/cpp/Esri/ArcGISRuntime/Toolkit/BasemapGalleryController.cpp:285]:
```suggestion
    The given default basemaps require either an access token or a user to be signed into the app.
```

File: uitools/toolkitwidgets/cpp/Esri/ArcGISRuntime/Toolkit/BasemapGalleryController.cpp [hunk 3]
```diff
@@ -415,7 +415,7 @@ namespace Esri::ArcGISRuntime::Toolkit {
     are fetched via \c{Portal::fetchBasemapsAsync}.
 
     This is useful for displaying an organization's basemaps or to display a gallery of the old-style basemaps
-    (which do not require an API key or named user.)
+    (which do not require an access token or named user.)
```
Comment (maryharvey) [inline uitools/toolkitwidgets/cpp/Esri/ArcGISRuntime/Toolkit/BasemapGalleryController.cpp:418]:
```suggestion
    (which do not require an access token or a user to sign in).
```

**Review by maryharvey — CHANGES_REQUESTED** [review]
@ldanzinger A few comments

**Comment by ldanzinger** [comment]
@maryharvey thanks, suggestions applied

**Review by dkemlage — COMMENTED** [review]
@ldanzinger 
A few possible minor word smith changes.

**Review by dkemlage — APPROVED** [review]
@ldanzinger LGTM. Approved.

**Pull request merged**